vaadin-tabs-flow-parent/vaadin-tabs-flow/src/main/java/com/vaadin/flow/component/tabs/TabSheetVariant.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.tabs;
+
+import com.vaadin.flow.component.shared.ThemeVariant;
+
+/**
+ * Set of theme variants applicable for {@code vaadin-tabsheet} component.
+ */
+public enum TabSheetVariant implements ThemeVariant {
+    //@formatter:off
+    LUMO_ICON_ON_TOP("icon-on-top"),
+    LUMO_CENTERED("centered"),
+    LUMO_SMALL("small"),
+    LUMO_MINIMAL("minimal"),
+    LUMO_HIDE_SCROLL_BUTTONS("hide-scroll-buttons"),
+    LUMO_EQUAL_WIDTH_TABS( "equal-width-tabs"),
```

Comment:
Hmm...should the variants only meaningful to the slotted `<vaadin-tabs>` be prefixed with `TABS`. For example `LUMO_TABS_CENTERED`?

Reply:
Yes, I think that makes sense.

Reply:
Done